README.md

```diff
@@ -26,9 +26,9 @@ both footprint and startup time are behaved pretty well.
 
 `Tangle-accelerator` as an intermediate server provides services like transaction explorer, issuing transfers and even PoW Accelerate to make attach to tangle faster. Certain API queries can store into memory cache for better searching and easier to reattach.
 
-```
+```shell
```

Comment:
Isn't it a graph composed of text characters instead of a shell script?

Reply:
I searched the list of supported markdown code block language on GitHub. Graph is not the one they support. I am still finding a good choice for the language here.

Reply:
Maybe just leave it empty?